LICENSE

```diff
@@ -1,25 +1,224 @@
-BSD 2-Clause License
-
-Copyright (c) 2017, Gamalon, Inc.
-All rights reserved.
-
-Redistribution and use in source and binary forms, with or without
-modification, are permitted provided that the following conditions are met:
-
-* Redistributions of source code must retain the above copyright notice, this
-  list of conditions and the following disclaimer.
-
-* Redistributions in binary form must reproduce the above copyright notice,
-  this list of conditions and the following disclaimer in the documentation
-  and/or other materials provided with the distribution.
-
-THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
-AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
-IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
-DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
-FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
-DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
-SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
-CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
-OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
-OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+The wrapper code is distributed under the Apache 2.0 license given below, while
+the original cephes code is distributed with the permission of the author,
+Stephen Moshier, with his original copyright as indicated by the email below.
+
+Delivered-To: thomasmarkovich@gmail.com
+Received: by 10.237.36.79 with SMTP id s15csp2466476qtc;
+        Sun, 30 Jul 2017 17:16:40 -0700 (PDT)
+Date: Sun, 30 Jul 2017 20:16:36 -0400 (EDT)
+From: Stephen Moshier <steve@moshier.net>
+To: Thomas Markovich <thomasmarkovich@gmail.com>
+Subject: Re: Cephes License Question
+
+Hi Thomas,
+
+I have looked up the Apache 2 license and it seems acceptable.  You might
```

Comment:
The email is a little non-committal. What I was hoping for was "I agree to distribute Cephes under the terms of the Apache License, version 2.0." The email does not resolve the following issue: while we may be "including the material in your work by permission of the author", the author (i.e. the copyright holder) does not explicitly grant us the right to, among other things, redistribute the material. I'm sorry but I really think we need a lawyer to take a look at this one.